Xsoar mirror saas (#29215)
demisto/content: Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring_description.md

## Configuration

> ### Server URL
> To get your server URL
>   - XSOAR 6.X: just use the URL of the server you are trying to connect to.
>   To access a Cortex XSOAR server which is in a Multi-tenant environment you should add the full path, i.e. https://xsoar_dns_address:8443/acc_MyTenant/
>   - XSOAR 8.X: Navigate to Setting > API Keys > Copy API URL

Reviewer: ```suggestion
### Obtaining the Server URL
- For XSOAR 6.x, simply use the URL of the server you wish to connect to.
  To access a Cortex XSOAR server in a multi-tenant environment, make sure to include the full path, such as: `https://xsoar_dns_address:8443/acc_MyTenant/`
- For XSOAR 8.x:
  1. Navigate to **Settings** > **API Keys** and click the **Copy API URL** button.
```

  - 
### API key
To gain the API key from Cortex XSOAR which you are connecting to:

>   - XSOAR 6.X: Navigate to Setting > API Keys > Get Your Key.
>   - XSOAR 8.X: Navigate to Setting > API Keys > +New Key.

Reviewer: ```suggestion
  - XSOAR 6.x: Navigate to **Setting** > **API Keys** > Click the **Get Your Key** button.
  - XSOAR 8.x: Navigate to **Setting** > **API Keys** > Click the **New Key** button.
```

### API Key ID

> This is required only when connecting to XSOAR 8.X tenants.
> After creating your API Key obtain its ID located in the ID column of the API Keys table.

Reviewer: ```suggestion
This parameter is required only when connecting to XSOAR 8.x tenants.
After creating your API Key, obtain its ID from the **ID** column of the API Keys table.
```
